lib/i18n_sonder/workers/upload_source_strings_worker.rb

```diff
@@ -0,0 +1,47 @@
+require 'sidekiq'
+require "mobility"
+
+module I18nSonder
+  module Workers
+    class UploadSourceStringsWorker
+      include Sidekiq::Worker
```

Comment:
I think by default sidekiq job retries if uncaught errors occur. Just wanted to call it out, in case it's not ideal to retry uploads.

Reply:
nah, part of the reason to make this a sidekiq worker was to retry the upload in case of intermittent failure. so this should be good.